docs/engines/trino.md

```diff
@@ -25,6 +25,10 @@ please refer to the documentation at [Iceberg Connector](https://trino.io/docs/c
 connector.name=arctic
 arctic.url=thrift://{ip}:{port}/{catalogName}
 ```
+- Configure the JVM configuration file for Trino in the {trino_home}/etc directory named `jvm.config` :
```

Comment:
Can you help to solve the format issue in line 23? It adds an empty line.

Reply:
Sure, done